kratos/geometries/tetrahedra_3d_4.h

```diff
@@ -506,9 +500,9 @@ template<class TPointType> class Tetrahedra3D4 : public Geometry<TPointType>
       double y30 = rP2[1] - rP3[1];
       double z30 = rP2[2] - rP3[2];
 
-      double detJMX = s10 * y20 * z30 + y10 * z20 * s30 + z10 * s20 * y30 - s30 * y20 * z30 - y30 * z20 * s10 - z30 * s20 * y10;
-      double detJMY = s10 * x20 * z30 + x10 * z20 * s30 + z10 * s20 * x30 - s30 * x20 * z30 - x30 * z20 * s10 - z30 * s20 * x10;
-      double detJMZ = s10 * x20 * y30 + x10 * y20 * s30 + y10 * s20 * x30 - s30 * x20 * y30 - x30 * y20 * s10 - y30 * s20 * x10;
+      double detJMX = s10 * y20 * z30 + y10 * z20 * s30 + z10 * s20 * y30 - s30 * y20 * z10 - y30 * z20 * s10 - z30 * s20 * y10;
```

Comment:
This is not the volume function. This calculates the circumradius of the tetrahedron for the quality function that need them. It had an error in one of the components of a diagonal that calculates another determinant. See ac707a565561248d556e0121634d586fe669350b. 

But I can rewrite the whole matrix so it looks like the volume if you want.